a10_octavia/controller/worker/tasks/server_tasks.py

```diff
@@ -28,7 +31,20 @@
 LOG = logging.getLogger(__name__)
 
 
-class MemberCreate(task.Task):
+class MemberBaseTask(task.Task):
+
+    def __init__(self, **kwargs):
```

Comment:
You don't need to include declare this explicity, when you can directly use, `BaseNetworkTask` from `a10_network_task`.